postcss.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  plugins: [
+    require('autoprefixer')({
+      browsers: ['> 5%']
```

Comment:
When I run the command line tool to list these browsers I only get these four:

    and_chr 53
    and_uc 11
    chrome 53
    ios_saf 10

If we change to 1 % we get this list instead:

    and_chr 53
    and_uc 11
    android 4.4.3-4.4.4
    android 4.4
    chrome 54
    chrome 53
    chrome 49
    edge 14
    firefox 49
    ie 11
    ios_saf 10
    ios_saf 9.3
    op_mini all
    safari 10
    samsung 4